content/docs/futures/spawning.md

```diff
@@ -300,14 +300,12 @@ tokio::run(lazy(|| {
             // Each spawned task will have a clone of the sender handle.
             let tx = tx.clone();
 
-            // In this example, "hello world" will be written to the
-            // socket followed by the socket being closed.
             io::read_to_end(socket, vec![])
-                // Drop the socket
```

Comment:
As of mentioned, the drop happens here. How would you feel about explicitly calling the drop fn to make it clearer?

Reply:
It would indeed be clearer.
On the other hand, it's not something one would need to write in real use cases and none of the other examples use `drop`.

The reason I moved the comment in the first place was that I noticed this piece of code was probably copied from the previous "Processing inbound sockets" code example which has the same comment right before `.map(|_| ())`. It was just a confusion.
I think it's best to just leave the comment as it was.

Reply:
Ok :+1: